src/actions/actionDuplicateSelection.tsx

```diff
@@ -275,6 +275,7 @@ const duplicateElements = (
       },
       getNonDeletedElements(finalElements),
       appState,
+      null,
```

Comment:
Why not to pass the instance here, action should have the access to it, right?

Reply:
ditto